python/ray/data/impl/block_batching.py

```diff
@@ -128,3 +138,23 @@ def _sliding_window(iterable: Iterable, n: int):
     for elem in it:
         window.append(elem)
         yield tuple(window)
+
+
+@ray.remote(num_cpus=0, placement_group=None)
+class _BlockPretcher:
+    """Helper actor that prefetches blocks asynchronously."""
+
+    def prefetch(self, *blocks) -> None:
+        pass
+
+
+def get_or_create_prefetcher() -> "ActorHandle":
+    node_id = ray.get_runtime_context().node_id
+    actor_name = f"dataset-block-prefetcher-{node_id}"
+    return _BlockPretcher.options(
+        scheduling_strategy=NodeAffinitySchedulingStrategy(node_id, soft=False),
+        name=actor_name,
+        namespace=PREFETCHER_ACTOR_NAMESPACE,
+        lifetime="detached",
```

Comment:
Could we make this a named, non-detached actor? If it's a detached actor, it will never get cleaned up.
```suggestion
```